docs/manual/common/guide/devmode/code/build-services-ports.sbt

```diff
@@ -4,6 +4,6 @@ lagomServicesPortRange in ThisBuild := PortRange(40000, 45000)
 
 //#service-port
 lazy val usersImpl = (project in file("usersImpl"))
-  .enablePlugins(LagomJava)
+  .enablePlugins(LagomScala)
```

Comment:
The problem here is that by doing that you are also changing the java documentation.

Reply:
I went too fast actually and thought that you were reporting the issue. Now I see that you sent a PR. I have the right fix and I will try to push to your branch so your PR is still valid.

Reply:
Actually, it won't work because your PR is based on Lagom 1.4.x. It must have been based on master and we then cherrypick on 1.4.x and 1.3.x. I will close this one and send a new PR. 

Thanks a lot for reporting it.